packages/dds/tree/src/core/schema-stored/schema.ts

```diff
@@ -226,6 +250,29 @@ export class ObjectNodeStoredSchema extends TreeNodeStoredSchema {
 		};
 	}
 
+	public override encodeV2(): TreeNodeSchemaDataFormatV2 {
+		const fieldsObject: Record<string, FieldSchemaFormat> = Object.create(null);
+		// Sort fields to ensure output is identical for for equivalent schema (since field order is not considered significant).
+		// This makes comparing schema easier, and ensures chunk reuse for schema summaries isn't needlessly broken.
```

Comment:
don't love that this code is duplicated with codec elsewhere. Any way to restructure to remove that?